wordpress/versions.json

```
@@ -1,4 +1,11 @@
 [
+  {
+    "ref": "HEAD",
+    "tag": "trunk",
+    "cacheable": true,
```

```suggestion
    "cacheable": false,
```

We should not cache "moving" targets. If `cacheable` is set to true, we will use Docker cache. In this case, Docker sees that the build arguments are the same and it will take all layers from its cache. It won't actually run any `git` commands, and we will end up having the old cached image.

---

In general, `cacheable: true` should only be set for stable things like tags which will never change.

---

good catch! I ran the script with the default args I think 😬